exposed-jodatime/src/main/kotlin/org/jetbrains/exposed/sql/jodatime/DateFunctions.kt

```diff
@@ -16,6 +16,19 @@ class Date<T : DateTime?>(val expr: Expression<T>) : Function<DateTime>(DateColu
     }
 }
 
+/** Represents an SQL function that extracts the time part from a given datetime [expr]. */
+class Time<T : DateTime?>(val expr: Expression<T>) : Function<DateTime>(DateColumnType(true)) {
```

Comment:
As I can see for time in `exposed-java-time` and `exposed-kotlin-datetime` related `LocalTime` class is used. Is it intentional not to use `org.joda.time.LocalTime` for jodatime?

Reply:
Thank you for pointing this out. I will exclude Jodatime from these changes for now and deal with it in a separate PR.